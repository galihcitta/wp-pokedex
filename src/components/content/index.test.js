import React from "react";
import Content from "./index";

let wrapper;

beforeEach(() => {
    wrapper = shallow(<Content data-test="content-test-wrapper" />);
});

describe("Header Menu Lang Component", () => {
    it("renders and create snapshot <Content />", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it("renders", () => {
        expect(wrapper.exists()).toBe(true);
    });
});
