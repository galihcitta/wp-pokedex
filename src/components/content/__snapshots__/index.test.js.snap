// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Menu Lang Component renders and create snapshot <Content /> 1`] = `
.c0 {
  background-color: #f7f7f7;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin: 0;
  padding: 0 15px;
  width: 100%;
}

<div
  className="c0"
  data-test="content-test-wrapper"
/>
`;
